addon/theming.js

```diff
@@ -0,0 +1,88 @@
+// Create a dummy theme profile based on the dark theme
+// NOTE: This should not be required once
+// https://bugzilla.mozilla.org/show_bug.cgi?id=1435216 is addressed
+const theme = {
+  colors: {
+    accentcolor: "#4A4A4F",
+    textcolor: "#fff",
+    popup: "#4A4A4F",
+    popup_text: "#fff",
+    popup_highlight: "#6D6D6F",
+    popup_highlight_text: "#fff"
+  }
+};
+
+const updateBackground = function(el, color) {
+  el.style.background = color;
+};
+
+const updateColor = function(el, color) {
+  el.style.color = color;
+};
+
+function updateSidebarStyle() {
+  if (theme.colors) {
```

Comment:
can these styles be applied with css instead? select a parent element el.classList.add('dark-theme') then style all the children within it dark?
We tlaked about how the border and carat of the panel can only be changed with JS, but in all other cases css is better